core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java

```diff
@@ -83,7 +84,7 @@ public static void main(String[] args) throws Exception {
 
   @Override
   public String keyword() {
-    return "create-empty";
+    return EXE_NAME;
```

Comment:
I'm not sure I see the benefit of this... it seems to just relocate the constant from an inline literal to a new line that's really only ever needed once (see my subsequent comment about why it's not needed for the test).